src/lib.rs

```diff
@@ -116,7 +116,15 @@ fn get_dependencies(blob_paths: &[&PathBuf]) -> HashMap<String, Vec<String>> {
             .expect("Could not convert to string.")
             .to_owned();
 
-        let buffer = fs::read(&path).expect("Could not read path.");
+        let buffer;
+        match fs::read(&path) {
+            Ok(r) => buffer = r,
+            Err(_) => {
+                eprintln!("Warning: Could not read file {}, skipping...", filename);
+                return;
+            },
+        }
```

Comment:
```suggestion
        match fs::read(&path) {
            Ok(b) => buffer = b,
            Err(_) => {
                eprintln!("Warning: Could not read file: {}", path.display());
                return;
            },
        }
```

Let's show the relative path instead of just the filename

Reply:
Done